graylog2-server/src/main/java/org/graylog2/plugin/journal/RawMessage.java

```diff
@@ -260,8 +274,14 @@ public String toString() {
         return helper.toString();
     }
 
+    public Object getMessageQueueId() {
+        return messageQueueId;
+    }
     public long getJournalOffset() {
```

Comment:
Should we annotate this as `@Deprecated` as well?

Reply:
Yes, I agree.